source/module.c

```diff
@@ -18,6 +18,7 @@
 #include "websocket.h"
 
 #include <aws/auth/auth.h>
+#include <aws/cal/cal.h>
```

Comment:
utterly trivial: why was this added?
```suggestion
```

Reply:
ah, can be removed. was experimenting with error mapping and forgot to get rid of it